crates/loam-cli/src/commands/dev/mod.rs

```diff
@@ -143,7 +135,7 @@ impl Cmd {
                                 && !parent_is_in_watched_dirs)
                             {
                                 println!("File changed: {path:?}");
-                                if let Err(e) = tx.blocking_send(Message::FileChanged) {
+                                if let Err(e) = tx.blocking_send("FileChanged".to_string()) {
```

Comment:
I actually prefer the enum with a single case over a string. For one it's less to send.

Reply:
Okay, I'll go with it.